addons/im_livechat/controllers/main.py

```diff
@@ -24,10 +24,9 @@ def livechat_lib(self, ext, **kwargs):
         if isinstance(mock_attachment, list):  # suppose that CSS asset will not required to be split in pages
             mock_attachment = mock_attachment[0]
         # can't use /web/content directly because we don't have attachment ids (attachments must be created)
-        status, headers, content = request.env['ir.http'].binary_content(id=mock_attachment.id, unique=asset.checksum)
-        content_base64 = base64.b64decode(content) if content else ''
-        headers.append(('Content-Length', len(content_base64)))
-        return request.make_response(content_base64, headers)
+        _, headers, content = request.env['ir.http'].binary_content(res_id=mock_attachment.id, unique=asset.checksum)
```

Comment:
I think we want to avoid `_` variables because it is overriding the translate function. Using `_status` if the goal is to avoid unused variable linter?